internal/armada/repository/queue.go

```diff
@@ -9,6 +11,8 @@ import (
 
 const queueHashKey = "Queue"
 
+var QueueNotFound = errors.New("Queue does not exist")
```

Comment:
The naming convention in go is to prefix well known errors with 'Err'. So 'ErrQueueNotFound' might be a better name to use here.